OWNERS

```diff
@@ -0,0 +1,79 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [
+        {
+          name: 'cramforce',
+          requestReviews: false
+        },
+        {
+          name: 'dvoytenko',
+          requestReviews: false
+        },
+        {
+          name: 'jridgewell',
+          requestReviews: false
+        }
+      ]
+    },
+
+    {
+      pattern: '**/validator-*.{protoascii,html,out}',
+      owners: [
+        {
+          name: 'ampproject/wg-caching',
+          notify: true
+        }
+      ]
+    },
+
+    {
+      pattern: '*.md',
+      owners: [
+        { name: 'ampproject/wg-outreach' }
+      ]
+    },
+
+    {
+      pattern: '{.*,babel.config.js,gulpfile.js}',
+      owners: [
+        { name: 'ampproject/wg-infra' }
+      ]
+    },
+
+    {
+      pattern: '{package.json,yarn.lock}',
+      owners: [
+        { name: 'ampproject/wg-infra' },
+        { name: 'ampproject/wg-runtime' },
+        { name: 'ampproject/wg-performance' }
+      ]
+    },
+
+    {
+      // The below rule will be here only temporarily during the transition
+      // period while many owners files are being updated as contributors and
+      // teams notice which files are out of date. The "notify" option means the
+      // owners bot will tag me on any PR that touches an owners file. This will
+      // allow me to verify syntax and semantics of all updates until the syntax
+      // check API for Travis to use is complete.
+
+      // Though this gives me the ability to approve changes to owners files,
+      // I do not currently have merge access; the owners check is currently
+      // non-blocking; and I run and maintain the owners bot application.
```

Comment:
nit: For more clarity, I suggest changing this paragraph from first person to third person (referencing `@rcebulko`).

Reply:
Done